### PR TITLE
NAS-127745 / 24.10 / Accessibility improvements for `Cron Scheduler`

### PR DESCRIPTION
**Pull request by RehanY147**
Check ticket for testing

### CODE REVIEW

File: src/app/modules/scheduler/components/scheduler-modal/scheduler-preview-column/scheduler-preview-column.component.html [hunk 1]
```diff
@@ -1,7 +1,13 @@
 <section class="preview-header">
   <h4 class="crontab-header">
     {{ 'Schedule Preview' | translate }}
-    <button class="close-btn" mat-icon-button ixTest="close-scheduler" [matDialogClose]>
+    <button
+      class="close-btn"
+      mat-icon-button
+      aria-label="Close scheduler"
```
Comment (denysbutenko) [inline src/app/modules/scheduler/components/scheduler-modal/scheduler-preview-column/scheduler-preview-column.component.html:7]: aria-labels need to be translatable

File: src/app/modules/scheduler/components/scheduler/scheduler.component.html [hunk 2]
```diff
@@ -13,33 +13,40 @@
...
+      </mat-option>
+      <mat-option
+        class="schedule-option"
+        [value]="customValue"
+        [ixTest]="[controlDirective.name, 'presets', customValue]"
+        (onSelectionChange)="onOptionSelectionChange($event)"
+      >
+        {{ 'Create' | translate }}
+        <span class="preset-description">
+          {{ 'Create custom schedule' | translate }}
```
Comment (denysbutenko) [inline src/app/modules/scheduler/components/scheduler/scheduler.component.html:49]: It looks weird having two `Create` words. I think we should keep only the first one.
image

**Review by denysbutenko — CHANGES_REQUESTED** [review]
Works well using keyboard navigation. Just a few minor things

File: src/app/modules/scheduler/components/scheduler-modal/scheduler-preview-column/scheduler-preview-column.component.html [hunk 3]
```diff
@@ -1,9 +1,16 @@
 <section class="preview-header">
   <h4 class="crontab-header">
     {{ 'Schedule Preview' | translate }}
-    <button class="close-btn" mat-icon-button ixTest="close-scheduler" [matDialogClose]>
-      <ix-icon name="cancel"></ix-icon>
-    </button>
+    <div class="close-wrapper" [attr.aria-label]="'Close scheduler' | translate">
```
Comment (denysbutenko) [inline src/app/modules/scheduler/components/scheduler-modal/scheduler-preview-column/scheduler-preview-column.component.html:4]: Other buttons has aria-label without a wrapper. Why is it needed here?

**Review by denysbutenko — APPROVED** [review]
Looks good

File: src/app/modules/scheduler/components/scheduler-modal/scheduler-preview-column/scheduler-preview-column.component.html (hunk 3 above)
Comment (RehanY147) [reply src/app/modules/scheduler/components/scheduler-modal/scheduler-preview-column/scheduler-preview-column.component.html:4]: Don't know why but it doesn't work on the button or icon itself. Only works on them if not used as an input property. But we need to use it as an input property to provide translation logic. So it works there if I use it like `aria-label="Close scheduler"` but not as `[attr.aria-label]="'Close scheduler' | translate"`

**Pull request merged**